theme/components/dropdown.scss

```diff
@@ -36,26 +27,15 @@
 }
 
 .ck-dropdown__panel {
-	// This is to get rid of flickering when the tooltip is shown under the panel,
-	// which looks like the panel moves vertically a pixel down and up.
-	-webkit-backface-visibility: hidden;
-
 	border: 1px solid ck-border-color( 'foreground' );
-	display: none;
-	position: absolute;
 	left: 0px;
```

Comment:
Shouldn't it also land in the Wireframe? OTOH, there could be a theme which positions the panel in a different way. Hard to tell.